### PR TITLE
variable added and constructor creater and getter setter and display …

### PR DESCRIPTION
**Pull request by mhshahib**
…method created

send a merge request and please review my code

### CODE REVIEW

File: src/burger/BurgerDetails.java [hunk 1]
```diff
@@ -1,4 +1,57 @@
 package burger;
 
 public class BurgerDetails {
+    private String nameOfBurger;
+    private String nameTopping;
+    private double priceOfBurger;
```
Comment (ayoubaba) [inline src/burger/BurgerDetails.java]: variable name will be priceOfTheBurger

Comment (mhshahib) [reply src/burger/BurgerDetails.java]: code is updated according to your variable name priceOfTheBurger

**Review by ayoubaba — APPROVED** [review]
now is fine and i will approved it

**Comment by trahman1** [comment]
i am merging your branch with master

**Pull request merged**